setup.py

```diff
@@ -52,7 +52,7 @@ def remove_bdist_rpm_source_file():
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.2.1',
+    'version': '2.2.2',
```

Comment:
that is another question. but I don't want 2 rpms with the same version and different checksums.